### PR TITLE
Implement menu accelerators

### PR DESCRIPTION
**Pull request by cra0zy**
Only implements it for Gtk side of things, WPF and Mac only have stubs.

### CODE REVIEW

File: Xwt/Xwt/Key.cs [hunk 1]
```diff
@@ -27,6 +27,7 @@
 
 namespace Xwt
 {
+	[Flags]
```
Comment (Therzok) [inline Xwt/Xwt/Key.cs]: \Why was this changed to `Flags`?

Comment (cra0zy) [inline Xwt/Xwt/Key.cs]: So multiple `Key` can be sent as accelerator, I could have also made `Accelerator` class to take in an array of `Key`, but that is really memory inefficient compared to this.

Comment (cra0zy) [inline Xwt/Xwt/Key.cs]: Then again most shortcuts only use one key and muliple modifiers... I can change it if you want.

Comment (Therzok) [inline Xwt/Xwt/Key.cs]: If I do `a | b`, it'll result in a totally other key, not combine the two keys. I'm possibly missing the point here, but having `Flags` won't make keys combinable.

Comment (cra0zy) [inline Xwt/Xwt/Key.cs]: Good point, I never looked into `Key` values and I always assumed it was a flag in Gtk like modifier keys... sorry about that.

Comment (Therzok) [inline Xwt/Xwt/Key.cs]: No worries, just wanted to know what was up. Glad we got it solved!
